domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientIT.java

```diff
@@ -1023,16 +1074,16 @@ public void saveBatchNullConnection(TestContext context) {
 
   @Test
   public void saveBatchNullList(TestContext context) {
-    createFoo(context).saveBatch(FOO, (List<Object>)null, context.asyncAssertSuccess(save -> {
-      context.assertEquals(0, save.getNumRows());
+    createFoo(context).saveBatch(BAR, (List<Object>)null, context.asyncAssertSuccess(save -> {
+      context.assertNull(save);
```

Comment:
This is an unneccessary breaking change that is inconsistent with vertx-pg-client that returns an empty RowSet when "UPDATE ... WHERE ..." does not update any record.
Can you return an empty RowSet?

```
  private RowSet<Row> emptyRowSet(String ... columnNames) {
    return new RowSet<Row>() {
      List<String> columnsNames = Arrays.asList(columnNames);

      @Override
      public int rowCount() {
        return 0;
      }

      @Override
      public List<String> columnsNames() {
        return columnsNames;
      }

      @Override
      public int size() {
        return 0;
      }

      @Override
      public <V> V property(PropertyKind<V> propertyKind) {
        return null;
      }

      @Override
      public RowSet<Row> value() {
        return this;
      }

      @Override
      public RowIterator<Row> iterator() {
        return new RowIterator<Row>() {
          @Override
          public boolean hasNext() {
            return false;
          }

          @Override
          public Row next() {
            return null;
          }
        };
      }

      @Override
      public RowSet<Row> next() {
        return null;
      }
    };
  }
```

Reply:
I have a FakeRowSet in test helpers.. which is this + some extra. That of course, have to be moved.

Reply:
Yes, good idea.